package NAlice;

message TFreshOptions {
    optional bool ForceEntireFresh = 1;
    repeated string ForceForForms = 2;
    repeated string ForceForEntities = 3;
    repeated string ForceForPrefixes = 4;
    // Deprecated. Only for granet backward compatibility.
    repeated string ForceForExperiments = 5;
}
